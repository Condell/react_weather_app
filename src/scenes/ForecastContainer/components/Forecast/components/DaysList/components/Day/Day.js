import React from "react";
import styled from "styled-components";
import PropTypes from "prop-types";

import Title from "../../../../../../../../components/Title/Title"
import { unixToDate } from "../../../../../../../../services/helpers"

const DayWrapper = styled.div`
  background-color: white;
  display: flex;
  justify-content: center;
`;

const Day = props => {
  const formattedDate = unixToDate(props.unixTime);
  const icon = props.weatherIcon;
  return (
    <DayWrapper {...props}>
      <img
        src={`http://openweathermap.org/img/w/${icon}.png`}
      />
      <h3>
        {formattedDate}
      </h3>
    </DayWrapper>
  );
};

export default Day;
