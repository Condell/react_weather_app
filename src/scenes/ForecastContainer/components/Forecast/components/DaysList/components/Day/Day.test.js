import React from "react";
import { shallow } from "enzyme";
import 'jest-styled-components'
import toJson from "enzyme-to-json"

import Day from "./Day";

jest.mock("./Day.js", () => "Day")

describe("Day", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Day />);
  });

  it("should render", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});