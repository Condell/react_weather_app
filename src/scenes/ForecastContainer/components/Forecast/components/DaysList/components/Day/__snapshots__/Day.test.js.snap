// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Day should render 1`] = `<Day />`;
